tests/e2e/sample-applications/mysql-persistent/storageclass/ocs-storagecluster-ceph-rbd.yaml

```diff
@@ -0,0 +1,22 @@
+allowVolumeExpansion: true
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    description: Provides RWO Filesystem volumes, and RWO and RWX Block volumes
+  name: ocs-storagecluster-ceph-rbd
```

Comment:
Same here. I would advise against applying this to a live cluster by adding a comment, or removing this from PR.

Reply:
https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html-single/deploying_openshift_data_foundation_using_amazon_web_services/index#:~:text=Verify%20that%20the%20following%20storage,openshift%2Dstorage.noobaa.io

Reply:
If this is for advanced write up, it shouldn't be in test/e2e dir

Reply:
but I'll let this one go :)